nvflare/lighter/study.py

```diff
@@ -0,0 +1,124 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+
+import argparse
+import json
+import os
+from datetime import datetime
+
+import yaml
+
+
+def get_input(prompt, item_list, multiple=False):
+    answer = input(prompt)
+    result = None
+    if multiple:
+        try:
+            if answer == "":
+                result = []
+            else:
+                result = [item_list[int(i)] for i in answer.split(",")]
+        except BaseException:
+            print("Input contains errors (non-integer or out of index range)")
+    else:
+        try:
+            result = item_list[int(answer)]
+        except ValueError:
+            print(f"Expect integer but got {answer.__class__.__name__}")
+        except IndexError:
+            print("Number out of index range")
+    if result is None:
```

Comment:
If result is None, then it means that user didn't select anything, right? We should print an error message before exit.

Reply:
A message is shown when result is an empty list in multiple=True.  When result is None (exception caught), users will be prompted to enter the answer again.